lib/matplotlib/backends/web_backend/mpl.js

```diff
@@ -125,6 +125,23 @@ mpl.figure.prototype._init_canvas = function() {
     // Throttle sequential mouse events to 1 every 20ms.
     rubberband.mousemove('motion_notify', mouse_event_fn);
 
+    function scroll_fn(event) {
+        event['data'] = 'scroll'
+        // http://www.adomas.org/javascript-mouse-wheel/
+        if (event.wheelDelta) { /* IE/Opera. */
+                event.step = event.wheelDelta/120;
+        } else if (event.detail) { /** Mozilla case. */
+            /** In Mozilla, sign of delta is different than in IE.
+             * Also, delta is multiple of 3.
+             */
+            event.step = -event.detail/3;
+        }
+        mouse_event_fn(event);
+    }
+
+    canvas_div[0].addEventListener('DOMMouseScroll', scroll_fn, false);
+    canvas_div[0].addEventListener('mousewheel', scroll_fn, false);
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/Events/mousewheel <- this seems rather down on 'mousewheel'.

`canvas_div.on('wheel mousewheel', scroll_fn)` causes the events to be triggered (I was wrong about what 'scroll' meant), but the events you get back are wrapped jquery events so of course none of the current code works correctly (it ends up passing `None` through for the position which causes the python to barf when you do math with it).